contracts/ccip/GovernanceCCIPRelay.sol

```diff
@@ -18,10 +18,9 @@ contract GovernanceCCIPRelay is IGovernanceCCIPRelay {
   address public immutable timelock;
 
   /// @inheritdoc IGovernanceCCIPRelay
-  address public destinationReceiver;
+  mapping(uint64 => address) public destinationReceivers;
 
-  /// @inheritdoc IGovernanceCCIPRelay
-  uint64 public destinationChainSelector;
+  uint64 private CCIPMainnetChainSelector = 5009297550715157269;
```

Comment:
@voith this can be declared as a `constant`
Also please rename it in **uppercase** to follow the Solidity styling guide.

Reply:
done